### DISCUSSION TITLE
REST API to change the token in Protection Bypass for Automation

### DISCUSSION THREAD

**Discussion by ammubhave** [vercel/community#7087] [Feature Question]
### Question

I was looking at the REST APIs and I couldn't find a way to change the token used in Protection Bypass for Automation programatically. This requires human intervention where someone has to login to the dashboard and update this value.

Is there a way to do this via the REST API?

**Comment by kitfoster** [comment]
Thanks for raising @ammubhave 🙏 

I've added the endpoint to our REST APIs (the change will be live soon), the endpoint is `PATCH /projects/:idOrName/protection-bypass`

It is also possible to do this using `protection_bypass_for_automation` in the `vercel_project` resource in the [Vercel Terraform provider](https://registry.terraform.io/providers/vercel/vercel/latest/docs)

**Comment by kitfoster** [comment]
Here is the REST API endpoint: [Update Protection Bypass for Automation](https://vercel.com/docs/rest-api/endpoints/projects#update-protection-bypass-for-automation)
